Dockerfile

```diff
@@ -13,7 +12,29 @@ COPY --chown=1001:0 . /tmp/src
 
 USER 1001
 
-ENV RAILS_ENV=production RAILS_LOG_TO_STDOUT=true QMAKE=/usr/lib64/qt5/bin/qmake DISABLE_ASSET_COMPILATION=true
+ENV RAILS_ENV=production RAILS_LOG_TO_STDOUT=true QMAKE=/usr/lib64/qt5/bin/qmake DISABLE_ASSET_COMPILATION=true BUNDLE_WITHOUT=development:test:tableau
+
+# Install the dependencies
+RUN /usr/libexec/s2i/assemble
+
+CMD ["/opt/app-root/src/entrypoint.sh"]
+
+FROM compliance-backend as compliance-tableau
+
+USER 0
+
+RUN subscription-manager attach && \
```

Comment:
@skateman you need to remove this , the node where it runs should be already subscribed, this logic must not be part of the image build process.